content/source/docs/enterprise/api/run.html.md

```diff
@@ -281,10 +297,13 @@ This endpoint queues the request to perform a discard; the discard might not hap
 
 This endpoint represents an action as opposed to a resource. As such, it does not return any object in the response body.
 
+-> **Note:** This endpoint is disabled for [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) or [team token](../users-teams-organizations/service-accounts.html#team-service-accounts).
```

Comment:
I think getting rid of the `...` in the payload would help the syntax parsing here.

Reply:
Ugh, this went all over the place in github. It's fine in our markdown processor, and I think having the ... provides value by making the example more legible... I'll see if adjusting it a bit can make Github hold its horses.

Reply:
Well, it worked in the main diff view but it's still grumpy in this comment view. :/ I'll take half a win, I guess.